### PR TITLE
v3.0.0-avalonia10-preview2

### PR DESCRIPTION
**Pull request by SKProCH**
Most likely latest version with Avalonia 0.10.x support
Version 3.0.0-rc0 was replaced with 3.0.0-avalonia10-preview1 (in order to keep version names clear)
- Bump Avalonia version dependency to 0.10.13
- Implement adapting to system base theme mode on Windows
- Improve/redesign Expander, TextBlock, Separator and other styles
- Create ContentExpandControl, used for animate resizing
- Add TransitioningContentControl style support
- Allow use Avalonia.Diagnostics to dialogs when using DEBUG profile
- Attach TextBlock properties to NumericUpDown
- Add padding to alert dialog and text field dialog
- Fix AutoCorrectPositionConverter #172
- Add default value for Slider, correct dots positioning
- Change resources apply logic to avoid colors apply delay while application starting
- Correct HsbConverter.ToColor method
- Add static part names pool
- Add auto-expand feature on NavigationDrawer
- Update contrast color algorithm
- Add InputMethod.IsInputMethodEnabled for TextBox

### CODE REVIEW

**Comment by SKProCH** [comment]
This PR merges all changes made before 3.0.0 branch started targeting Avalonia 11

File: Material.DataGrid/Material.DataGrid.csproj [hunk 1]
```diff
@@ -21,8 +21,8 @@
     </ItemGroup>
 
     <ItemGroup>
-      <PackageReference Include="Avalonia" Version="0.10.11" />
-	  <PackageReference Include="Avalonia.Controls.DataGrid" Version="0.10.11" />
+      <PackageReference Include="Avalonia" Version="0.10.13" />
+	  <PackageReference Include="Avalonia.Controls.DataGrid" Version="0.10.13" />
```
Comment (appleneko2001) [inline Material.DataGrid/Material.DataGrid.csproj:25]: Have some strange extra spacing in this line
not such critical point but recommend remove it to keep code clean

File: Material.Styles/Themes/SystemThemeProbe.cs [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+using Material.Styles.Themes.Base;
+
+namespace Material.Styles.Themes;
+
+public static class SystemThemeProbe {
+    /// <summary>
+    /// Tries to resolve base theme mode for the current system
+    /// </summary>
+    /// <returns>Base theme (<c>Dark</c>/<c>Light</c>) or <c>null</c></returns>
+    public static BaseThemeMode? GetSystemBaseThemeMode() {
```
Comment (appleneko2001) [inline Material.Styles/Themes/SystemThemeProbe.cs:13]: SystemThemeProbe can be implemented for other OS too, not just for Windows I think.
Anyway it is a good attempt to let library choose which theme scheme are suitable by OS user settings.

I think I should note this as a part of 3.0.0 major update to improve this feature

**Review by appleneko2001 — APPROVED** [review]
ok

File: Material.Styles/Themes/SystemThemeProbe.cs (hunk 2 above)
Comment (SKProCH) [reply Material.Styles/Themes/SystemThemeProbe.cs:13]: I think what we should do everything related to system theme probing in dev branch (and also other features, what can be done independently of Avalonia 11)

**Pull request merged**